README.markdown

```diff
@@ -22,35 +22,48 @@ If you want to hack on `CocoaPods.app`:
+$ cd CocoaPods-app
+$ gem install rest
+$ rake app:prerequisites --quiet
+$ open app/CocoaPods.xcworkspace
 ```
 
+> Having an issue with running `rake app:rerequisities --quiet`? You might need to fork the CocoaPods/Specs repository locally to your computer. To do this simply run the following command:
+>
+> ```sh
+> $ git clone git@github.com:CocoaPods/Specs.git ~/.cocoapods/repos/master --depth=1
+> ```
+
+> Having an issue with `ruby/config.h` not found on OS X Mojave/Catalina? You need to install the SDK Headers by running the following:
+>
+> ```sh
+> $ open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
+> ```
+
 This will set up your environment with a compiled versions of: ruby, git, cocoapods (the gem), hg, openssl, etc into both `destroot` and `workbench`.
 
 ### Rake Tasks
 
 The main tasks can be found with `rake -T`:
 
-```
-rake app:build               # Build release version of application
-rake app:clean               # Clean
-rake app:prerequisites       # Prepare all prerequisites for building the app
-rake app:update_version      # Updates the Info.plist of the application to reflect the CocoaPods version
-rake bundle:build            # Build complete dist bundle
-rake bundle:clean:all        # Clean all artefacts, including downloads
-rake bundle:clean:artefacts  # Clean build and destroot artefacts
-rake bundle:submodules       # Ensure Submodules are downloaded
-rake bundle:test             # Test bundle
-rake bundle:verify_linkage   # Verifies that no binaries in the bundle link to incorrect dylibs
-rake release                 # Create a clean release build for distribution
-rake release:build           # Perform a full build of the bundle and app
-rake release:cleanbuild      # Create a clean build
-rake release:sparkle         # Version bump the Sparkle XML
-rake release:upload          # Upload release
+```sh
+$ rake app:build               # Build release version of application
+$ rake app:clean               # Clean
+$ rake app:prerequisites       # Prepare all prerequisites for building the app
+$ rake app:update_version      # Updates the Info.plist of the application to reflect the CocoaPods version
+$ rake bundle:build            # Build complete dist bundle
+$ rake bundle:clean:all        # Clean all artefacts, including downloads
+$ rake bundle:clean:artefacts  # Clean build and destroot artefacts
+$ rake bundle:submodules       # Ensure Submodules are downloaded
+$ rake bundle:test             # Test bundle
+$ rake bundle:verify_linkage   # Verifies that no binaries in the bundle link to incorrect dylibs
+$ rake release                 # Create a clean release build for distribution
+$ rake release:build           # Perform a full build of the bundle and app
+$ rake release:cleanbuild      # Create a clean build
+$ rake release:sparkle         # Version bump the Sparkle XML
+$ rake release:upload          # Upload release
```

Comment:
Adding the `$` means you can't triple click a line to copy & paste for the CLI with little up-side: IMO not worth it. Thanks for thinking about how to improve it though!